src/renderer/api-decorator.js

```diff
@@ -416,7 +412,7 @@ limitations under the License.
                     onContentReady(window, cb);
                 }
             },
-            getUuid: generateGuidSync,
+            getUuid: process.getGUID,
```

Comment:
Where is this coming from? A quick search in core didn't bring anything up.

Reply:
@StevenEBarbaro Added this to the runtime recently: 

https://github.com/openfin/runtime/blob/develop/atom/renderer/atom_sandboxed_renderer_client.cc#L93

Reply:
👍